NewForce-Cohort-7/tabloidfullstack-numpty-buggers
# Js comment

Description
This pull request is for TabloidFullStack Trello ticket number 19: (https://trello.com/c/Xb25oljg/19-19-view-a-posts-comments), View a Post's Comment 
DISCRIPTION OF TICKET: when clicking on a a post's details, the user can  choose to view the comments which are rendered in order of date created. 

The branch for the pull request: js-comment

Updates
View comments by Post 

Type of change
New Feature - non-breaking

Testing Instructions
Follow these instructions to protect the branch you are working on and to test this pull request: https://gist.github.com/jordan-castelloe/96f76d5bd4cfbd2181084af674c2a1af#when-you-need-to-test-a-teammates-pr

Seed The Data
If you do not currently have the code for this project, clone the repository - LINK TO REPOSITORY

Open the TabloidFullStack.sln in visual studio and prepare your database.
In the github repository: LINK TO THE REPO
Open the SQL folder: lINK TO SQL FOLDER
there are two files - LINK TO FILES
In Visual Studio in the top navbar click View and select SQL Server Object Explorer. Open the server marked as "SQLEXPRESS" in the SQL Server Object Explorer on the left side of Visual Code. Once there, add a new SQL server by selecting the 'Add SQL Server' option at the top of the side bar. Copy the data from the seed data and execute by pressing the green Play button.
start the server by clicking the solid green play button in the middle top of your screen. A browser window will display with the Tabloid landing page.


Test view comments by post Id

NUMBERED STEP BY STEP DIRECTIONS
Once in the branch js-comment and having booted up VS and performed "npm start" at the tabloid folder you should have swagger and localhost:3000 pulled up, login as [[[foo@bar.com](mailto:foo@bar.com)] (no password needed)
Click on "Posts" in the navbar and you should see a list of all the posts.
Click on details on the third post down (or any post with comments) 
Select The View Comments button
Confirm that the list of comments that appear is for the right post
